### PR TITLE
OF-1868: Cluster isMessageCarbonsEnabled and hasRequestedBlocklist to other nodes

### PR DESCRIPTION
**Pull request by mightymop**
isMessageCarbonsEnabled and hasRequestedBlocklist have to be clustered to other members too.

### CODE REVIEW

**Comment by mightymop** [comment]
Also check this: https://github.com/igniterealtime/openfire-hazelcast-plugin/pull/31

File: xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java [hunk 1]
```diff
@@ -68,6 +72,18 @@ public String getActiveList() {
     public boolean isOfflineFloodStopped() {
         return offlineFloodStopped;
     }
+    
+    /**
```
Comment (GregDThomas) [inline xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java]: Could you remove the comment? Not really necessary as git will show where/when it was added. Thanks!

File: xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java [hunk 2]
```diff
@@ -68,6 +72,18 @@ public String getActiveList() {
...
+    * Added for OF-1868
+    */
+    public boolean hasRequestedBlocklist() {
+        return hasRequestedBlocklist;
+    }
+    
+    /**
+    * Added for OF-1868
```
Comment (GregDThomas) [inline xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java]: (this comment too)

File: xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java [hunk 3]
```diff
@@ -81,6 +97,12 @@ public void writeExternal(ObjectOutput out) throws IOException {
             ExternalizableUtil.getInstance().writeSafeUTF(out, activeList);
         }
         ExternalizableUtil.getInstance().writeBoolean(out, offlineFloodStopped);
+
```
Comment (GregDThomas) [inline xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java]: Ditto

File: xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java [hunk 4]
```diff
@@ -94,5 +116,11 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
             activeList = ExternalizableUtil.getInstance().readSafeUTF(in);
         }
         offlineFloodStopped = ExternalizableUtil.getInstance().readBoolean(in);
+        
+        /**
+        * Added for OF-1868
```
Comment (GregDThomas) [inline xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java]: Ditto

**Comment by GregDThomas** [comment]
It looks to me like you're committing all your changes to master; there are changes in here now unrelated to OF-1868. You generally need to work on branches - you might want to have a look at https://guides.github.com/introduction/flow/ if you're unfamiliar with how branching works.

**Comment by mightymop** [comment]
Ok so u wont commit the code until i splitted it into several branches? What should i do now? Btw. I have the codechanges running for 6 month  now and they r running quite good.

**Comment by deleolajide** [comment]
> Ok so u wont commit the code until i splitted it into several branches?

You don't need to create several branches. Just a single one with all the changes concerning this issue OF-1868. That way it won't be affected by other stuff happening on your master repo :-)

**Comment by mightymop** [comment]
I opened up the pull request from a new branch now:

https://github.com/igniterealtime/Openfire/pull/1534

**Pull request closed without merging**